sublime_music/ui/common/spinner_image.py

```diff
@@ -34,10 +34,14 @@ def set_from_file(self, filename: Optional[str]):
             filename = None
         self.filename = filename
         if self.image_size is not None and filename:
-            pixbuf = GdkPixbuf.Pixbuf.new_from_file_at_scale(
-                filename, self.image_size, self.image_size, True
-            )
-            self.image.set_from_pixbuf(pixbuf)
+            try:
+                pixbuf = GdkPixbuf.Pixbuf.new_from_file_at_scale(
+                    filename, self.image_size, self.image_size, True
+                )
+                self.image.set_from_pixbuf(pixbuf)
+            except:
+                print("Warning: could not load", filename, ". "
+                      "Probably not an image. \"Artwork not found\"?")
```

Comment:
Use `logging`, not `print`, and also include the actual exception

```suggestion
                logging.warn(f"could not load {filename}. Probably not an image. {e}")
```